app/code/Magento/CustomerGraphQl/Model/Resolver/GenerateCustomerToken.php

```diff
@@ -14,24 +14,43 @@
 use Magento\Framework\GraphQl\Query\ResolverInterface;
 use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
 use Magento\Integration\Api\CustomerTokenServiceInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
 
 /**
  * Customers Token resolver, used for GraphQL request processing.
  */
 class GenerateCustomerToken implements ResolverInterface
 {
+    /**
+     * Configuration path to Customer Token Lifetime  setting
+     */
+    const TOKEN_LIFETIME_PATH_KEY = 'oauth/access_token_lifetime/customer';
+
+    /**
+     * Max GraphQl Integer value
+     */
+    const MAX_INT = 2147483647;
```

Comment:
I would not recommend delegating this responsibility to the resolver somehow. If the value in the database is bigger than the INT size (the system allows you to set it more than INT size), the resolver will return an incorrect value for the expiration time. Moreover, the system operates with the `access_token_lifetime` as with the Float type.